### DISCUSSION TITLE
Vod hls transcoding Error

### DISCUSSION THREAD

**Discussion by oguzhangokboru** [ant-media/Ant-Media-Server#5512] [General]
I'm using Ant Media Enterprise Edition. I have an issue with vod transcoding. I installed vod_transcode.sh into the server and set up everything. But transcoding for long videos isn’t working.

For example, if i upload a 1 minute video, the outputs are:
vod_id.mp4
vod_id.m3u8
vod_id_360p
vod_id_360p.m3u8
vod_id_480p
vod_id_480p.m3u8
vod_id_720p
vod_id_720p.m3u8

Everything works fine as you can see.
But if i upload a 30min, one hour or two hours long video, the outputs are:
vod_id.mp4
vod_id_360p
vod_id_480p
vod_id_720p

Here, no m3u8 files are being generated. I tried so so many times but it's not working. Please help me get this issue fixed ASAP. 
Also Ant Media Dashboard doesn’t allow uploading video more than one. How to upload video files in bulk and make ./vod_transcode.sh transcode all the the files automatically?

Kind regards,

**Comment by Mohit-3196** [comment]
Hi All,
We are discussing this on GitHub.
FYI all https://github.com/ant-media/Ant-Media-Server/issues/5513

Thank you
